tests/unittests/test_handler/test_handler_reset_rmc.py

```diff
@@ -0,0 +1,54 @@
+from cloudinit.config import cc_reset_rmc
+
+from cloudinit import util
+
+from cloudinit.config.cc_write_files import write_files
+from cloudinit.tests import helpers as t_help
+from cloudinit.tests.helpers import mock
+
+import logging
+import os
+
+LOG = logging.getLogger(__name__)
+NODE_FILE = '/etc/ct_node_id'
+RSCT_PATH = '/opt/rsct/install/bin'
+
+
+class TestRsctNodeFile(t_help.FilesystemMockingTestCase):
+    def setUp(self):
```

Comment:
i think you can drop this setUp now, as you're not (i think)  using the tmpdir that you create.